digits/model/views.py

```diff
@@ -96,6 +95,21 @@ def customize():
             'snapshot': snapshot
             })
 
+@blueprint.route('/extension-static/<extension_id>/<path:filename>')
```

Comment:
does it have to be in the **model** views? Can you move this one level up maybe?